color.cpp

```diff
@@ -87,6 +87,13 @@ std::string to_str(const color& in_color) noexcept
   return msg;
 }
 
+bool is_first_color_winner(const color& c1, const color& c2)
+{
+    double hue1 = calc_hue(c1);
+    double hue2 = calc_hue(c2);
+
+    return (hue1 == 0.0 && hue2 == 120.0) || (hue1 == 120.0 && hue2 == 240.0) || (hue1 == 240.0 && hue2 == 0.0);

```

Comment:
This was all that was needed to pass the test, well done.
Sidenote : there will be more tests 😸.